### PR TITLE
(node/auxtel-hcu01.cp.lsst.org) add el9 network and tests

### PR DESCRIPTION
**Pull request by dtapiacl**
Converting auxtel devices to EL9.

### CODE REVIEW

**Review by gseriche — APPROVED** [review]

File: hieradata/node/auxtel-hcu01.cp.lsst.org.yaml [hunk 1]
```diff
@@ -1,21 +1,55 @@
...
+      ethernet: {}
+      ipv4:
+        method: "disabled"
+      ipv6:
+        method: "disabled"
+  enp8s0:
+    content:
+      connection:
+        id: "enp8s0"
+        uuid: "df30450e-88e5-40c5-bdc8-0edf49cd7e3e"
```
Comment (jhoblitt) [inline hieradata/node/auxtel-hcu01.cp.lsst.org.yaml]: duplicate uuid

File: spec/hosts/nodes/auxtel-hcu01.cp.lsst.org_spec.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        it_behaves_like 'nm manual interface'
+        it { expect(nm_keyfile['ipv4']['address1']).to eq('192.168.1.1/24') }
+      end
+
+      context 'with enp7s0' do
+        let(:interface) { 'enp7s0' }
+
+        it_behaves_like 'nm disabled interface'
+        it_behaves_like 'nm ethernet interface'
+        it { expect(nm_keyfile['ipv6']['method']).to eq('disabled') }
```
Comment (jhoblitt) [inline spec/hosts/nodes/auxtel-hcu01.cp.lsst.org_spec.rb]: This duplicates one of the examples in `nm disabled interface`.

File: spec/hosts/nodes/auxtel-hcu01.cp.lsst.org_spec.rb [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      end
+
+      context 'with enp8s0' do
+        let(:interface) { 'enp8s0' }
+
+        it_behaves_like 'nm disabled interface'
+        it_behaves_like 'nm ethernet interface'
+        it { expect(nm_keyfile['ipv6']['method']).to eq('disabled') }
```
Comment (jhoblitt) [inline spec/hosts/nodes/auxtel-hcu01.cp.lsst.org_spec.rb]: This duplicates one of the examples in `nm disabled interface`.

File: spec/hosts/nodes/auxtel-hcu01.cp.lsst.org_spec.rb [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+      context 'with enp7s0' do
+        let(:interface) { 'enp7s0' }
+
+        it_behaves_like 'nm disabled interface'
+        it_behaves_like 'nm ethernet interface'
+        it { expect(nm_keyfile['ipv6']['method']).to eq('disabled') }
+      end
+
+      context 'with enp8s0' do
```
Comment (jhoblitt) [inline spec/hosts/nodes/auxtel-hcu01.cp.lsst.org_spec.rb]: Two disabled interfaces listed out is OK but if it was any more I'd have preferred to see an array iterated over to keep it DRY.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

**Comment by jhoblitt** [comment]
Something to be aware of is that interfaces named "in the bios". E.g. eno1, em1, etc. will be stable when upgrading to EL9 but the automagically named interfaces will change.

**Review by jhoblitt — APPROVED** [review]

**Pull request closed without merging**